lib/offsite_payments/integrations/molpay.rb

```diff
@@ -60,8 +60,7 @@ def form_fields
         end
 
         def amount=(money)
-          #Molpay minimum amount is 1.01
-          if money.is_a?(String) or money.to_f < 1.01
+          if money.is_a?(String)
```

Comment:
@saifiikmal I believe the suggestion was to be more explicit and use `unless money.is_a?(Money) || money <= 0` vs. `if money.is_a?(String)`